src/scripts/main.js

```diff
@@ -1,5 +1,73 @@
 'use strict';
 
+function css(element, style) {
+  for (const property in style) {
+    element.style[property] = style[property];
+  }
+}
+
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const _isAxisX = (axis) => axis === 'x';
+const maxCoordinates = {
+  minX: 0,
+  maxX: wall.clientWidth - spider.width,
+
+  minY: 0,
+  maxY: wall.clientHeight - spider.height,
+};
+
+function validateAxisValue(axis, value) {
+  const { minX, maxX, minY, maxY } = maxCoordinates;
+  const isAxisX = _isAxisX(axis);
+
+  const result = isAxisX ? {
+    isCorrect: minX <= value && value <= maxX,
+  } : {
+    isCorrect: minY <= value && value <= maxY,
+  };
+
+  if (!result.isCorrect) {
+    return {
+      ...result,
+      newCoordinate: isAxisX
+        ? value < minX ? minX : maxX // new X coordinate
+        : value < minY ? minY : maxY, // new Y coordinate
```

Comment:
is code require comments - you have to rewrite it in the way, that it doesn't require comments